### DISCUSSION TITLE
Community Collaborative Project

### DISCUSSION THREAD

**Discussion by naomi-lgbt** [EddieHubCommunity/support#819] [Ideas]
In today's community call, an idea was proposed that we start a new collaborative project within the organisation. This discussion is going to serve as a Single Source of Truth for collecting ideas and feedback on what kind of project we'd like to build.

**Comment by Vyvy-vi** [comment]
I'd like to point out some of the projects that need work:

- [CommunityAPI](https://github.com/EddieJaoudeCommunity/api):
  This is an API for collecting our community data from github and discord and we plan on linking a lot of projects and there are many ideas further on for with this-
   - [EddieBotUI](https://github.com/EddieJaoudeCommunity/EddieBotUI): (This is an old project, that needs to be revamped) The old community stats project had hard-coded values... And this could be deployed, updated with live stats and with a better UI.
   - [EddieMeter](https://github.com/EddieJaoudeCommunity/support/issues/511): (idea popped up in a discussion about badges) A github action that updates badges with data from our community that members can put on their github profile. A version of this is just normal shields.io badges... discussed here, which we might build upon it: https://github.com/EddieJaoudeCommunity/support/discussions/758
   - [EddieMap/EddieBotLive](https://github.com/EddieJaoudeCommunity/EddieBotLive): (This is an old project, that needs some docs and maybe more customisation)
   - Updates to [EddieBot](https://github.com/EddieJaoudeCommunity/EddieBot) Introduce stats and bio synced with the new community API
- [EddieJaoudeCommunity site](https://github.com/EddieJaoudeCommunity/EddieJaoudeCommunity.github.io): More ideas and more features. Also, a couple of issues open. (Svelte)
- [gh-actions-table-generater](https://github.com/EddieJaoudeCommunity/gh-actions-html-table-generator/issues/13): This needs to be bumped to TypeScript.
- [Template repo](https://github.com/EddieJaoudeCommunity/template): This needs more templates and a github action that can sync all the community repos with these templates(ISSUE_TEMPLATES and PR_TEMPLATES).
- [EddieHub Branding](https://github.com/EddieJaoudeCommunity/Branding): Repo for community art and for ideas about community etc. (This needs an action, that can take all the uploaded images in the repo, and convert those to links embedded in a markdown file).

A few ideas that I'd like to pitch:

- askEddie: FAQ website(or a page in our community website) that contains loads of Eddie's insight and questions Eddie answered, which might be useful for everyone.
- A gh-action that posts to discord [#first-timers](https://discord.gg/zjGcnEzfgC) ~~(and maybe Twitter on a TwitterBot)~~ everytime a new discussion is opened on support repo under [First Timers](https://github.com/EddieJaoudeCommunity/support/discussions/categories/first-timers) section.
- Kick MEE6 and NightBot and implement that all in EddieBot :joy:
- Community Blog/something(It keeps updates about the community and events in form of posts/blog on a community website, which kinda acts like a showcase for the community)

**Reply by eddiejaoude** [reply]
Thank you @Vyvy-vi for highlighting existing projects ❤️ !!

**Comment by Nitesh-thapliyal** [comment]
Some suggestion about the Collaborative project:
- we can create a repo name collaboration where folks can add their projects and others can collaborate and contribute to the project.
- we can work on the Hackathon projects that we created in MLH try to make it great

**Reply by eddiejaoude** [reply]
I like this idea, I am not sure people would want their project in a mono repo though? may be in this `collaboration` repo, it would be a list/table of projects and links to their repos?

**Reply by Vyvy-vi** [reply]
Perhaps we could add a COLLABORATION_PROJECTS.md in the `Hackathons` repo, with tables for project data and links to the project repos?

**Comment by ujjwal-kr** [comment]
What about a Twitter bot that retweets #EddieHub tweets?

**Reply by Vyvy-vi** [reply]
I think that's already a thing 🤔 
*tho I'm not sure if it retweets all the tweets
~~and anyways Eddie does that~~

**Reply by ujjwal-kr** [reply]
yeah automating the process is always better!

**Reply by Vyvy-vi** [reply]
https://twitter.com/Edbeepbot
I think @kcoder63 ([@starlightknown](https://twitter.com/starlightknown) on twitter) (Karuna on discord) made the bot 🤔

**Reply by starlightknown** [reply]
Yes i did

**Reply by Vyvy-vi** [reply]
Is it still running?

**Reply by starlightknown** [reply]
Not daily, not after lhd . We can make some changes to it if you like
